docs/concepts/models/sql_models.md

```diff
@@ -1,37 +1,40 @@
 # SQL models
 
-SQL models are the main types of models used by SQLMesh. SQL models consist of three sections:
+SQL models are the main type of models used by SQLMesh. These models can be defined using either SQL or Python that generates SQL.
 
-* `MODEL` DDL
-* Optional sql statements
-* The model's `SELECT` statement
+## SQL-based definition
 
-SQL models are designed to look and feel like you're simply using SQL, but they can be customized for advanced use cases.
+The SQL-based definition of SQL models is the most common one, and consists of the following sections:
 
-## Definition
-To create an example SQL model, add a file named `my_model.sql` into the `models/` directory (or a subdirectory of `models/`) within your SQLMesh project.
+* The `MODEL` DDL
+* Optional pre-statements
+* A single query
+* Optional post-statements
 
-Although the name of the file doesn't matter, it is recommended to name it something that includes the word "model". Each file can only have one model defined within it.
+These models are designed to look and feel like you're simply using SQL, but they can be customized for advanced use cases.
+
+To create a SQL-based model, add a new file with the `.sql` suffix into the `models/` directory (or a subdirectory of `models/`) within your SQLMesh project. Although the name of the file doesn't matter, it is recommended to use a name that preserves the model's name. Each file can only have one model defined within it.
```

Comment:
Instead of "it is recommended...", maybe "it is customary to use the model's name (without the schema) as the file name. For example, the file containing the model `sqlmesh_example.seed_model` file would be named `seed_model.sql`."